rllib/policy/torch_policy.py

```diff
@@ -329,12 +323,32 @@ def compute_log_likelihoods(
     @DeveloperAPI
     def learn_on_batch(
             self, postprocessed_batch: SampleBatch) -> Dict[str, TensorType]:
-        # Get batch ready for RNNs, if applicable.
-        pad_batch_to_sequences_of_same_size(
-            postprocessed_batch,
-            max_seq_len=self.max_seq_len,
-            shuffle=False,
-            batch_divisibility_req=self.batch_divisibility_req)
+        if self.config["_use_trajectory_view_api"]:
```

Comment:
Would be great to have separate methods instead of if/else (then this method just dispatches to the two different helper methods).

Reply:
done